### ISSUE TITLE
process spawner in appconfig assumes that project is being run from manage.py

### ISSUE THREAD

**Issue by jMyles** [hangarunderground/tiempo#18]
Here: 

https://github.com/hangarunderground/tiempo/blob/03dad102478b0db905d3897708e933abacd1d2ba/tiempo/contrib/django/appconfig.py#L18

...we are assuming that the user's cwd is the same as manage.py.  It might not be.

**Comment by vepkenez** [comment]
Good point.
My assumption is actually that we are running using the _hx_ command which _does_ require that you re in the manage.py location... but still.

**Comment by jMyles** [comment]
I've been trying to figure out a way to either spawn a new process based on
django.core.management.call_command or by using django_admin.py, which
pretty much puts us back at the beginning of needing to run
settings.configure().

I think we need a completely different approach.

**Comment by vepkenez** [comment]
I don't know... The management command is kind of nice because you could
also run it manually...

**Comment by jMyles** [comment]
Agreed - let's keep the management command.  However, it is not a solution to multiprocessing generally. I'm not super comfortable just running settings.configure() for the user and assuming we got everything right.

Let's - for now - just require that they provide a self-contained script to run.

The problem of running an arbitrary python callable in a separate process is a big one - bigger than the scope of tiempo, and certainly bigger than our immediate need.

**Comment by vepkenez** [comment]
What do you mean by 'a self contained script to run'?

**Comment by vepkenez** [comment]
Maybe you mean how celery does it.
http://celery.readthedocs.org/en/latest/django/first-steps-with-django.html
